src/components/Input/Input.scss

```
@@ -0,0 +1,10 @@
+@import "@config/variables";
+
+.input {
+  width: 300px;
+  height: 84px;
+}
+
+.input:focus {
+  border: 1px solid $main-color;
+}
```

scss позволяет делать так, надо этим пользоваться
```suggestion
.input {
  width: 300px;
  height: 84px;
  
  &:focus {
  border: 1px solid $main-color;
}
}
```

То есть селектор ниже строится на основании селектора выше